src/UI/Pages/CompileReports/FilterWidget/CompileReportsFilterWidget.tsx

```diff
@@ -17,27 +16,15 @@ export const CompileReportsFilterWidget: React.FC<Props> = ({
   setFilter,
 }) => {
   const [filterKind, setFilterKind] = useState<CompileReportParams.Kind>(
-    CompileReportParams.Kind.Result
+    CompileReportParams.Kind.Status
   );
 
-  const updateResult = (success?: boolean) => setFilter({ ...filter, success });
-
-  const compileStatuses = Object.keys(CompileReportParams.CompileStatus).map(
-    (k) => CompileReportParams.CompileStatus[k]
-  );
-
-  const updateCompileStatus = (selectedCompileStatuses: string[]) =>
+  const updateCompileStatus = (selectedCompileStatus: string | null) =>
     setFilter({
       ...filter,
-      status:
-        selectedCompileStatuses.length > 0
-          ? selectedCompileStatuses.map(
-              (compileStatus) =>
-                CompileReportParams.CompileStatus[
-                  compileStatus.replace(/\s+/g, "")
-                ]
-            )
-          : undefined,
+      status: selectedCompileStatus
+        ? CompileStatus[selectedCompileStatus.replace(/\s+/g, "")]
```

Comment:
This does not do anything. 😅 
If it's a type problem, we should explicitly fix this.

Reply:
It removes the space from `in progress` so the enum member is used

Reply:
Yea I know what it does 😅 . I mean, the only possible options are the ones from the enum... So why go from enum value to enum key to enum value... Thats what I mean with it doesn't do anything.
If this is in there because of a type issue, the type cast should be done explicitly.

Reply:
The enum values are used as display names, and the keys are used internally, and they are not exactly the same, so it's an actual lookup and not only a cast. If you prefer we can use a string literal type instead of an enum and not have this distinction

Reply:
Yea thats probably the reason why I never create enums. String literal types are more forgiving and simpler to use and they are equally safe. But we have plenty of enums in the codebase. So no need to change this now.